agent_api_rest/src/issuance/credential_issuer/credential.rs

```diff
@@ -266,10 +266,10 @@ mod tests {
 
     #[rstest]
     #[case::without_external_server(false, false, 0)]
-    #[case::with_external_server(true, false, 0)]
-    #[case::with_external_server_and_self_signed_credential(true, true, 0)]
-    #[should_panic(expected = "assertion `left == right` failed\n  left: 500\n right: 200")]
-    #[case::should_panic_due_to_timout(true, false, DEFAULT_EXTERNAL_SERVER_RESPONSE_TIMEOUT_MS + 100)]
+    // #[case::with_external_server(true, false, 0)]
+    // #[case::with_external_server_and_self_signed_credential(true, true, 0)]
+    // #[should_panic(expected = "assertion `left == right` failed\n  left: 500\n right: 200")]
+    // #[case::should_panic_due_to_timout(true, false, DEFAULT_EXTERNAL_SERVER_RESPONSE_TIMEOUT_MS + 100)]
```

Comment:
All these test as well as any other failing test in `agent_api_rest` can be fixed simply by setting the new config variable to true: `credential_offer_by_value_enabled: true` in the `test-config.yaml` file. 

So no other alterations to any of the tests in `agent_api_rest` are needed since setting the config variable to `true` is basically already the status quo when it comes to these tests :)